extension/vsc-js/test/suite/index.js

```diff
@@ -0,0 +1,42 @@
+const path = require('path');
```

Comment:
tabs here

Reply:
these are auto-generated files, should I still remove the tabs?

Reply:
Yes please